schemas/author.json

```diff
@@ -85,13 +94,19 @@
 			"type": "string",
 			"description": "An optional URL to this contributor's webpage.",
 			"validation": [
-				"url"
+				"url",
+				"nullable"
 			]
 		},
 		"userGroupId": {
 			"type": "integer",
 			"description": "in which user group should this contributor be identified. Usually one of the author roles, such as Author or Translator.",
 			"apiSummary": true
+		},
+		"userGroupLabel": {
+			"type": "string",
+			"description": "Usually one of the author roles, such as Author or Translator.",
+			"apiSummary": true
```

Comment:
Add the `"readOnly": true` property to this, since it is not something we can write to the database.

Reply:
Added